scripts/linters/general_purpose_linter.py

```diff
@@ -232,6 +232,17 @@
             'typings/guppy-defs-b5055b963fdbea5c6c1e92dbf58fdaf3ea0cd8ba.d.ts',
             'core/templates/services/UpgradedServices.ts'),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(
+            r'eslint-(disable|enable)(-next-line)?'
```

Comment:
Don't think you need all the surrounding stuff -- just have a check for 'no-explicit-any'? That will catch more stuff.

Reply:
done